### DISCUSSION TITLE
App crashing

### DISCUSSION THREAD

**Discussion by Jooglypoof** [NightscoutFoundation/xDrip#3471] [General]
The latest auto update from the xdrip+ app is causing the app the crash as soon as I open it and my phone tells me this (see photo). I downgraded to a previous version so it wouldn't crash but I thought I should let someone know.
![Screenshot_20240520_134811_Device care](https://github.com/NightscoutFoundation/xDrip/assets/170294108/1757b1db-c991-495e-80e7-3db0b7826555)

**Comment by Navid200** [comment]
Would you please try different versions and identify exactly which one is the very first that causes this?
https://navid200.github.io/xDrip/docs/ReleaseNotes.html

**Reply by Navid200** [reply]
Also, please tell us what phone brand and model this is happening on and what Android version you have on it.

**Reply by Jooglypoof** [reply]
Hey Navid, I fixed the problem myself so it's all good. I just restarted my phone and it fixed itself so false alarm. Sorry for any inconvenience.

**Reply by Navid200** [reply]
Oh no worries.
This is the best outcome.  Wonderful

**Reply by Navid200** [reply]
And thanks for resporting
